security/src/main/java/zone/cogni/lib/security/common/SecurityConfiguration.java

```diff
@@ -69,6 +76,10 @@ static class BasicAuthCondition {
     static class Saml2Condition {
     }
 
+    @ConditionalOnProperty(name = "cognizone.security.auth-method", havingValue = "disabled")
+    static class DisabledCondition {
```

Comment:
huh?

Reply:
kewl spring stuff 😛 
to support generic config with property and @EnableSecurityXXXX, I convert the property to a Configuration which goes back to @EnableSecurityXXXX
Then I use condition classes to find the good one or fail if value is wrong.

I will add a bit of comment about what happens